emails/models.py

```diff
@@ -195,20 +195,28 @@ def fxa_locale_in_premium_country(self):
     def avatar(self):
         return self.fxa.extra_data.get("avatar")
 
-    @cached_property
+    @property
     def relay_addresses(self):
-        # TODO: Remove cached_property, to avoid cache invalidation
         return RelayAddress.objects.filter(user=self.user)
 
-    @cached_property
+    @property
     def domain_addresses(self):
-        # TODO: Remove cached_property, to avoid cache invalidation
         return DomainAddress.objects.filter(user=self.user)
 
     @property
     def num_active_address(self):
         return len(self.relay_addresses) + len(self.domain_addresses)
 
+    @property
+    def total_masks(self) -> int:
+        return len(self.relay_addresses) + len(self.domain_addresses)
+
+    @property
+    def at_mask_limit(self) -> bool:
+        if self.has_premium:
+            return False
+        return len(self.relay_addresses) == settings.MAX_NUM_FREE_ALIASES
```

Comment:
Please use ``.count()`` instead of ``len(queryset)`` so that the DB return is just a number, not all the `RelayAddress` rows

```suggestion
        return self.relay_addresses.count() == settings.MAX_NUM_FREE_ALIASES
```